pages/support/_enterprise-support.html

```diff
@@ -4,11 +4,9 @@ <h2>Enterprise Support</h2>
     <p>
       Gruntwork also offers an enterprise support plan that includes:
       <ul>
-        <li>Live chat in phone/video calls</li>
-        <li>Design reviews</li>
-        <li>Code reviews</li>
-        <li>Prioritized bug fixes</li>
-        <li>Guaranteed same-day response times</li>
+        <li>SLA on response times</li>
```

Comment:
We could still list our offerings here, e.g. "Next- or same-day response time SLA"